sourcegraph/about
# Sitemap config

This closes #5711 and provides a sitemap config to generate our `sitemap.xml` and `robots.txt` files for SEO purposes.

### Changelog
- added `next-sitemap` to dev deps
- added sitemap config
- added to post build script

Note: Remember to run `yarn ci` locally to have the latest deps.

### Test
1. Ensure prettier has standardized the proposed changes.
2. Run `yarn build` and check the `out` directory for a `robots.txt` and `sitemap.xml` file
3. Check the preview to see if you can load those files at the root `/robots.txt` and `/sitemap.xml`
